src/CheckoutSdk/Webhooks/PartialUpdateWebhookRequest.cs

```diff
@@ -0,0 +1,39 @@
+using System.Collections.Generic;
+
+namespace Checkout.Webhooks
+{
+    /// <summary>
+    /// Defines a <see cref="PartialUpdateWebhookRequest"/>.
+    /// </summary>
+    public class PartialUpdateWebhookRequest : Webhook
+    {
+        /// <summary>
+        /// Creates a new <see cref="PartialUpdateWebhookRequest"/> instance.
+        /// This class is suitable for partially updating an existing webhook.
+        /// </summary>
+        /// <param name="webhook">The base webhook.</param>
+        public PartialUpdateWebhookRequest(IWebhook webhook)
+        {
+            if (!string.IsNullOrEmpty(webhook.Url)) Url = webhook.Url;
+            Active = webhook.Active;
+            if (!(webhook.Headers == null)) Headers = webhook.Headers;
+            if (!(webhook.EventTypes == null)) EventTypes = webhook.EventTypes;
```

Comment:
```suggestion
            if (webhook.Headers != null) Headers = webhook.Headers;
            if (webhook.EventTypes != null) EventTypes = webhook.EventTypes;
```

This expression can be simplified a little